dashboard/src/features/shared/components/SearchBar/SearchBar.scss

```diff
@@ -76,9 +75,26 @@
     cursor: pointer;
   }
 }
+.viewdocs {
+  background-image: url('assets/images/info.png');
+  background-repeat: no-repeat;
+  background-size: 15px 15px;
```

Comment:
If we're using a 15x15 image, let's use a 30x30 variant for the 2x size instead of 64x64.